package.json

```diff
@@ -47,7 +47,7 @@
     "qunitjs": "^1.23.0",
     "raw-loader": "^0.5.1",
     "requirejs": "^2.2.0",
-    "simple-style-loader": "~0.2.0",
+    "simple-style-loader": "~0.3.0",
```

Comment:
Why don't you just get rid of the tilde? We should maintain this version and keep it in sync with the last tested published version known to work.